README.md

```diff
@@ -42,7 +44,8 @@ Find a bug or want to request a new feature?  Please let us know by submitting a
 Esri welcomes contributions from anyone and everyone. Please see our [guidelines for contributing](https://github.com/esri/contributing).
 
 ## Licensing
-Copyright 2013 Esri
+
+Copyright 2018 Esri
```

Comment:
Is it correct to change this to the current year? My [understanding](https://www.copyrightlaws.com/copyright-notice-year/) is that the year to include in a copyright notice is the year of first publication. Should this be `Copyright 2013-2018 Esri`?

Reply:
I was most certain it is the most recent date, but I will find out

Reply:
I got clarification on the copyright policy and updated it here http://devinfo.esri.com/sites/DeveloperCentral/github/SitePages/Policies.aspx

Thanks for pointing this out.

Reply:
You're welcome. Thank you for looking into it.